src/main/java/com/conveyal/gtfs/validator/MTCValidator.java

```diff
@@ -39,6 +42,32 @@ public void validate() {
             validateFieldLength(trip, trip.trip_headsign, 120);
             validateFieldLength(trip, trip.trip_short_name, 50);
         }
+
+        // Validate that calendars apply to at least one day of the week.
+        for (Calendar calendar : feed.calendars) {
+            validateCalendarDays(calendar);
+        }
+    }
+
+    /**
+     * Checks that a {@link Calendar} entity is applicable for at least one day of the week
+     * (i.e. at least one of the fields for Monday-Sunday is set to '1').
+     * @param calendar The {@link Calendar} entity to check.
+     * @return true if at least one field for Monday-Sunday is set to 1, false otherwise.
+     */
+    public boolean validateCalendarDays(Calendar calendar) {
+        boolean result = calendar.monday == 1
+            || calendar.tuesday == 1
+            || calendar.wednesday == 1
+            || calendar.thursday == 1
+            || calendar.friday == 1
+            || calendar.saturday == 1
+            || calendar.sunday == 1;
+
+        if (!result) {
+            if (errorStorage != null) registerError(calendar, SERVICE_WITHOUT_DAYS, "[Service applies to no day of the week.]");
```

Comment:
the badValue arg doesn't add any information here to the error description, so it should not be included.